### DISCUSSION TITLE
Side Chaining Question

### DISCUSSION THREAD

**Discussion by Loupap** [apohl79/audiogridder#536] [General]
Hello there,

I am new to Audiogridder and I am about to test it out today using Cubase 11 Pro as my DAW for the first time.

Quick question about sidechaining a plugin:

Will I be able to sidechain a plug in that is being activated through audiogridder on the slave computer?

Thanks,
Loupap

**Comment by apohl79** [comment]
Yes, but you need version 1.2.

**Reply by Loupap** [reply]
Yup thank you it does work.

However, I can't control the plugin from the slave via the screen capture? 

Am I missing something.

The plugin being used on the slave via AudioGridder is Trackspacer....which the sidechaining is working great.

**Reply by apohl79** [reply]
Make sure you have the permissions granted as needed: https://audiogridder.com/server/#install

**Reply by Loupap** [reply]
Yes thank you I figured that out shortly after I sent the message.

Do I need to worry about the buffer size in AG or is that controlled by my Daw Buffer Size.

When I mix off of my DAW I leave my Buffer at the Max 2048 in Cubase just due to the size of my projects.

**Reply by apohl79** [reply]
As long as you see no drop outs or something, you can keep the buffer size in ag at 0 blocks - means no additional buffering. One block is one daw i/o buffer (2k in your case).

**Comment by jaacMusic** [comment]
I also have a question about side-chain. I just created a sent on a track to another track with AG plugin. I don't seem to trigger the input signal on the destination track. Maybe I'm missing something but I can't see to find in an AG plugin chain how to tell which VST to receive the send input to trigger a side-chain compressor.

**Reply by apohl79** [reply]
When you activate the SC signal for the AG plugin, it forwards it to the server and all plugins of the AG chain will receive it. If a plugin does not support SC signals, it is supposed to ignore that channel.

**Reply by jaacMusic** [reply]
Hi Andreas, 
Sorry, I don't see to be able to forward SC signal to an AG plugin in another track. To test, I also added the same  VST on that destination track (VST can take SC signal: Waves F6 Stereo). When I flip from AG to the locally loaded Waves F6 Stereo it works but the one loaded in the AG plugin doesn't react at all. I tried different VST that supports SC and I wasn't able to accomplish it.
By the way, I use Cakewalk as a DAW on Windows 10 and my AG server is also a Windows 10 system.
Thanks!
